gc/base/OMRVMInterface.cpp

```diff
@@ -81,6 +81,15 @@ GC_OMRVMInterface::getOmrHookInterface(MM_GCExtensionsBase *extensions)
 }
 
 /**
+ * Return the OMR private hook interface for the Memory Manager
+ */
+J9HookInterface**
+GC_OMRVMInterface::getOmrPrivateHookInterface(MM_GCExtensionsBase *extensions)
+{
```

Comment:
I need a simple way to access all of hook interfaces from javadump code like below, 
any suggestion for access omr private hook interface? Thanks,

j9JavaVM->memoryManagerFunctions->j9gc_get_omr_hook_interface(j9JavaVM->omrVM)
j9JavaVM->memoryManagerFunctions->j9gc_get_omr_private_hook_interface(j9JavaVM->omrVM)
j9JavaVM->memoryManagerFunctions->j9gc_get_hook_interface(j9JavaVM)

j9JavaVM->internalVMFunctions->getVMHookInterface(j9JavaVM)
zip_getVMZipCachePoolHookInterface((J9ZipCachePool *) j9JavaVM->zipCachePool)
j9JavaVM->internalVMFunctions->getJITHookInterface(j9JavaVM)